python/cudf/dataframe/series.py

```diff
@@ -556,6 +566,12 @@ def to_array(self, fillna=None):
         """
         return self._column.to_array(fillna=fillna)
 
+    def _invert(self):
+        """Internal convenience function for inverting masked array
+        """
+        gpu_mask = self.to_gpu_array()
+        cudautils.invert_mask(gpu_mask, gpu_mask)
```

Comment:
I'm not sure this should go into `Series` - I don't think `invert` is usually a meaningful construct for a `Series`.

Reply:
Do you have suggestions on where this *should* go ?

Reply:
@quasiben I think it would make more sense for this to live at the Column level as opposed to the Series since it could in theory be used against an index as well.

Reply:
👍